src/test/resources/Logout.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Logout
+
+  An a user
+  I want to Log out
+  So I can use the app
+
+  Scenario: User can log out
+    Given User login with account as "John Doe" and password as "ThisIsNotAPassword"
```

Comment:
thay bằng
`user login with account as "John Doe" and password as "ThisIsNotAPassword"`
hành động giống nhau, thì nên dùng các step có sẵn, để đỡ phải sinh nhiều step mất thời gian